dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -755,6 +763,27 @@ void CommandBarFlyoutCommandBar::EnsureFocusedPrimaryCommand()
     }
 }
 
+void CommandBarFlyoutCommandBar::MoreButtonClicked(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
+{
+    // If the button get's invoked, we switch between secondary commands visible and secondary commands not visible
+    m_secondaryCommandsExpectedToBeVisible = !m_secondaryCommandsExpectedToBeVisible;
+}
+
+void CommandBarFlyoutCommandBar::OverFlowPopupClosed(const winrt::IInspectable& sender, const winrt::IInspectable& args)
+{
+    // We expected our secondary commands to be visible, yet the flyout was closed.
+    // Only scenario for this is when a secondary command was invoked, which means the complete flyout should close.
```

Comment:
>Only [](start = 7, length = 4)

What about when you hit the Esc key or close the flyout via some other light dismiss technique

Reply:
Other light dismiss methods work as expected, that is they take down the complete CommandBarFlyout, but also leave us in a clean state.